### ISSUE TITLE
Support for Windows or Powershell?

### ISSUE THREAD

**Issue by hkusulja** [lilydjwg/pssh#128]
Do you plan to implement/build pssh support for Windows command prompt or PowerShell 7 ?

**Comment by lilydjwg** [comment]
No, I don't know much about Windows and this project is basically in maintenance mode. But what doesn't work there anyway?

**Comment by lilydjwg** [comment]
I don't know what's the preferred way to run Python scripts on Windows, but if you install Python there you can try `python.exe setup.py install` and see what Python does for you.
